test/topology_custom/test_long_join.py

```diff
@@ -27,16 +27,20 @@ async def test_long_join(manager: ManagerClient) -> None:
     await asyncio.gather(task)
 
 @pytest.mark.asyncio
-async def test_long_join_drop_wntries_on_bootstrapping(manager: ManagerClient) -> None:
+async def test_long_join_drop_entries_on_bootstrapping(manager: ManagerClient) -> None:
```

Comment:
I hope the test still reproduces the original issue it was created for. Generally I think changing the test like that is risky -- unless you explicitly verified that it continues to catch the original issue by un-fixing it and seeing that the test fails. Otherwise I'd probably prefer to have a separate test for this new issue.

Reply:
Original problem and this one are similar. The reason is the same. It is simply that original issue is hit first and you need more then 2 nodes to hit new one if the original one is not hit. Creating separate test will make test.py run longer. Isn't this a killer argument?

Reply:
If this test covers both issues then of course it's fine to have a single test. I'm just worried if the first issue is still covered. (Is it?)

Reply:
It is.